.github/workflows/release.yml

```diff
@@ -87,23 +87,35 @@ jobs:
           asset_content_type: application/gzip
 
   release-docker:
-    name: Build and Push Semgrep Docker Container
+    name: Build and push Semgrep Docker image
     runs-on: ubuntu-latest
     if: ${{ !contains(github.ref,'-test-release') }}
     steps:
-      - uses: actions/checkout@v3
-        with:
-          submodules: true
```

Comment:
docker/build-push-action automatically uses the git repo content from the push event: https://github.com/docker/build-push-action#git-context